<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1788</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonCamera1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCamera2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera2</string>
    </property>
   </widget>
   <widget class="QVideoWidget" name="widgetCamera1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QVideoWidget" name="widgetCamera2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCamera">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>1001</width>
      <height>871</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTestQRCodDetection">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test QR code detection</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_1">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_2">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_3">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_4">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_5">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>480</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_6">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>590</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_7">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>700</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_8">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>810</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_9">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_10">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_11">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_12">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>370</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_13">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>480</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_14">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>590</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_15">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>700</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCircle_16">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>810</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1788</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
